README.md

```diff
@@ -116,6 +117,55 @@ An almost comprehensive example is:
 res0: util.Try[MyClass] = Success(MyClass(1,AdtB(1),List(1.0, 0.2),Map(key -> value),None))
 ```
 
+## Defining naming conventions
+
+In case the naming convention you use in your source configuration files differs
+from the one used in the objects you're loading the configs into, PureConfig
+allows you to define proper mappings. That configuration should be done by an
+implicit `ConfigFieldMapping` that should be in scope when loading or writing
+configurations. The `ConfigFieldMapping` trait has a single `apply` method that
+maps field names in Scala objects to field names in the source configuration
```

Comment:
I agree it sounds better to just say "config file" or "configuration file". I've fixed it in ea38720.